integr8ly/integreatly-operator
# add support for BU contact via envVar and RHMI CR

# Description

We need to contact BU to notify customer API usage, so this PR adds the ability to include BU email address to an envVar or via  RHMI CR 

https://issues.redhat.com/browse/MGDAPI-200

## Verification steps

1) Export an envVar variable with the BU email 
```bash
BU_ALERTING_EMAIL_ADDRESS=noreply-bu-test@rhmi-redhat.com
```
2) Install operator locally from this branch
3) Verify if the bu email is present in the RHMI CR
```bash
oc get rhmi rhmi -o json -n redhat-rhmi-operator | jq .spec.alertingEmailAddresses

{
  "businessUnit": "noreply-bu-test@rhmi-redhat.com",
  "cssre": ""
}

```

## Type of change
